README.md

```diff
@@ -7,5 +7,5 @@
 ## Descripcion:
 Saber previsión tiempo de un día 
```

Comment:
Os recuerdo que tenéis que describir, como en el juego de rol, dos cosas: quién es el cliente y qué problema tiene.

Reply:
Gracias profesor ahora lo hago

Reply:
Te agradezco la diligencia pero uno de los objetivos de aprendizaje del nivel 0 es que todo el mundo sepa hacer un PR. En [los objetivos de la sesión se hoy](https://github.com/JJ/MPDA-IS-ed/blob/main/sesiones/02.md#antes-de-esta-clase-deseable) está eso (también cambiar el avatar ;-) ) así que quizá Vicky o Ramil puedan hacer esas correcciones, sacando una rama de esta y haciendo un PR sobre feature-readme.md que el resto del equipo pueda examinar.
Podéis internamente (por el grupo de Telegram) asignar diferentes partes, o diferentes ficheros, a diferentes personas, para que no haya conflictos.